sdk/textanalytics/Azure.AI.TextAnalytics/src/Transforms.cs

```diff
@@ -32,6 +32,22 @@ internal static TextAnalyticsError ConvertToError(TextAnalyticsErrorInternal err
             return new TextAnalyticsError(errorCode, message, target);
         }
 
+        internal static TextAnalyticsError ConvertToError(DocumentError error)
```

Comment:
From this code, it looks like you are only using the `error.Error` property, so instead of passing the whole `DocumentError` object, consider sending to this function just `DocumentError.Error`.

Context: `DocumentError` has 2 main properties: the `id` of the document and the `error` itself. This function is already called per each document, which is why you never use `id` in here.

Reply:
updated